### ISSUE TITLE
Page language is not identified - icons.brand.uiowa.edu

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa801#9733]
# Motivation

Siteimprove is reporting "Page language is not identified". This is a WCAG 2.1 A compliance issue related to icons.brand.uiowa.edu.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r4

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/1216671/A11y/Page?pageId=75147585775&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r4/failed/bUxHZFRpVEI=;ODdkOTM5OWM=

Example
- https://icons.brand.uiowa.edu/

**Page language has not been identified**
Issue
3.1.1: Language of Page
The main language of the page has not been declared within the <html> element.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

 **Discussion:**
 - do we need an overall health check

**Proposed solution or definition of done:**
